core/modules/path/path.inc

```diff
@@ -184,6 +184,12 @@ function path_clean_string($string, array $options = array()) {
   elseif (!empty($options['langcode'])) {
     $langcode = $options['langcode'];
   }
+  if ($langcode == LANGUAGE_NONE) {
+    // Paths for language neutral content get transliterated according to
+    // current language.
+    global $language;
+    $langcode = $language->langcode;
```

Comment:
Would this work instead? It makes me uncomfortable that the language in which the interface is displayed would affect how a path is created. This would instead check the default language, which in the case of single-language sites, would result in the same outcome.

```suggestion
    $langcode = config_get('system.core', 'language_default');
```

Reply:
`language_default` - That's what @olafgrabienski suggested, but it has side effects. It could be the wrong choice on a multilingual site with language neutral content types.

`global $language` That's exactly how D7 Transliteration module does it. And no one seemed to have trouble with it the past years.

https://git.drupalcode.org/project/transliteration/-/blob/7.x-3.x/transliteration.inc#L182

Reply:
At the very least, we should add an inline comment with what you said @indigoxela 👍 

```suggestion
    global $language;
    // We are intentionally not using config_get('system.core', 'language_default')
    // here. That can have unexpected behavior in cases such as on a multilingual
    // site with language-neutral content types.
    $langcode = $language->langcode;
```